packages/app-management-service/src/manager/impl/BaseAppManager.ts

```diff
@@ -26,6 +26,16 @@ export abstract class BaseAppManager {
       fs.unlinkSync(filepath);
     }
   }
+
+  updatePageStatus(code: string, status: 'draft' | 'published'): void {
```

Comment:
probably better to export this union as a type variable so if needed somewhere else we would import it